SMPyBandits/Policies/GreedyOracle.py

```diff
@@ -0,0 +1,34 @@
+from .IndexPolicy import IndexPolicy
+import numpy as np
+
+class GreedyPolicy(IndexPolicy):
+    """ Greedy Policy for rotting bandits.
+    Selects arm with best last value.
+    Reference: [Heidari , Kearns & Roth, 2016]
+    """
+    def __init__(self, nbArms):
+        super(GreedyPolicy, self).__init__(nbArms)
+        self.arms_history = [np.array([]) for arm in range(nbArms)]
+
+    def getReward(self, arm, reward):
+        super(GreedyOracle, self).getReward(arm, reward)
+        self.arms_history[arm] = np.insert(self.arms_history[arm], 0, 0) + reward
+
+    def computeAllIndex(self,arms):
+        """ Compute the mean of the h last value """
+        for i, arm in enumerate(arms):
+            self.index[i] = self.arms_history[arm][0]
```

Comment:
- `h` is not used here: only one (`h=1`) sample is used!
- why store the whole history if only the last sample is used?

Reply:
yes , i think it is a wrong copy paste from SWA and I actually never used  this policy.